guides/getting-started.md

```diff
@@ -79,13 +79,14 @@ Whichever approach you use, be sure to import the Angular Material modules _afte
 
 Including a theme is **required** to apply all of the core and theme styles to your application.
 
-To get started with a prebuilt theme, include the following in your app's index.html:
-
-```html
-<link href="../node_modules/@angular/material/prebuilt-themes/indigo-pink.css" rel="stylesheet">
+To get started with a prebuilt theme, include one of Angular Material's prebuilt themes globally
+in your application. If you're using the Angular CLI, you can add this to your `styles.css`:
+```css
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
 ```
 
-Note that your app's project structure may have a different relative location for your node_modules.
+If you are not using the Angular CLI, you can include a prebuilt theme via a `<link>` element in
+your `index.html`.
```

Comment:
can we add back the example for this? or are you afraid people won't read and will just add both?